src/nomad_simulations/schema_packages/basis_set.py

```diff
@@ -967,10 +967,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.mt_r_min = None
 
-    @check_normalized
-    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
-        super().normalize(archive, logger)
-        # TODO: add spherical specification, once supported in `Grid`
```

Comment:
Note that this normalization has a decorator (as does the initializer). They are here to enforce a specific execution of the normalizer. I'd be cautious removing them here. Let us double-check this case (pls remind me).